### PR TITLE
[MOV] web, base_import: mobile tests should be in mobile suite

### PR DESCRIPTION
**Pull request by adr-odoo**
This fix some inconsistent error on runbot due to this commit:
https://github.com/odoo/odoo/commit/da7b59045d246c159f93bc75e3f19c6b1221d31c

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/base_import/static/tests/import_buttons_tests.js [hunk 1]
```diff
@@ -107,50 +106,4 @@ QUnit.test('kanban without import button', function(assert) {
...
-    assert.expect(1);
-
-    var list = createView({
-        View: ListView,
-        model: 'foo',
-        data: this.data,
-        arch: '<tree><field name="foo"/></tree>',
-        config: {
-            device: {
-                size_class: config.device.SIZES.XS,
```
Comment (adr-odoo) [inline addons/base_import/static/tests/import_buttons_tests.js]: I hesitate to force size_class in https://github.com/odoo-dev/odoo/blame/12.0-mobile-tests-fix-adr/addons/web/views/webclient_templates.xml#L585

File: addons/web/static/tests/helpers/test_utils.js [hunk 2]
```diff
@@ -438,7 +438,7 @@ function addMockEnvironment(widget, params) {
         initialConfig.device = _.clone(config.device);
         if ('device' in params.config) {
             _.extend(config.device, params.config.device);
-            config.device.isMobile = config.device.size_class <= config.device.SIZES.VSM;
+            config.device.isMobile = config.device.size_class <= config.device.SIZES.SM;
```
Comment (adr-odoo) [inline addons/web/static/tests/helpers/test_utils.js]: we can remove this line.

File: addons/base_import/static/tests/import_buttons_tests.js (hunk 1 above)
Comment (aab-odoo) [reply addons/base_import/static/tests/import_buttons_tests.js]: no need actually, as the code only checks the isMobile property

File: addons/web/static/tests/helpers/test_utils.js (hunk 2 above)
Comment (aab-odoo) [reply addons/web/static/tests/helpers/test_utils.js]: you must, i'd say

File: addons/base_import/static/tests/import_buttons_tests.js (hunk 1 above)
Comment (aab-odoo) [reply addons/base_import/static/tests/import_buttons_tests.js]: forget what i said, i agree that it should be forced as well as isMobile

File: addons/base_import/static/tests/import_buttons_mobile_tests.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    list.destroy();
+});
+
+QUnit.test('import button should be hidden in kanban on mobile screens', function (assert) {
+    assert.expect(1);
+    var done = assert.async();
+
+    // the kanban view is async in mobile mode, because it has to load an
+    // external additional library
+    testUtils.createAsyncView({
```
Comment (adr-odoo) [inline addons/base_import/static/tests/import_buttons_mobile_tests.js]: not async anymore, @aab-odoo such a bad reviewer :p

Comment (aab-odoo) [reply addons/base_import/static/tests/import_buttons_mobile_tests.js]: :+1:

**Comment by adr-odoo** [comment]
Linked to https://github.com/odoo/enterprise/pull/2962

**Pull request closed without merging**